test/test-framework/go.mod

```diff
@@ -5,10 +5,10 @@ go 1.13
 require (
 	github.com/operator-framework/operator-sdk v0.0.0
 	github.com/spf13/pflag v1.0.5
-	k8s.io/api v0.0.0
-	k8s.io/apimachinery v0.0.0
+	k8s.io/api v0.17.2
+	k8s.io/apimachinery v0.17.2
 	k8s.io/client-go v12.0.0+incompatible
-	sigs.k8s.io/controller-runtime v0.4.0
+	sigs.k8s.io/controller-runtime v0.5.1
 )
```

Comment:
We should not do these updates. See that the master branch is using the old versions
https://github.com/operator-framework/operator-sdk/blob/master/go.mod#L62

Reply:
Revert the go.mod and go.sum here.

Reply:
thanks for pointing this out :+1: 
removed